services.md

```diff
@@ -108,8 +119,11 @@ Generally, services shouldn't know most other services unless necessary.
 
 Consumer services know about source services through configuration (i.e. through the environment): no server names or URLs in a service's codebase
 
+**EXAMPLE:** In the MonoRail if a property changes we [publish an event](https://github.com/HouseTrip/HouseTrip-Web-App/blob/master/app/observers/event_observer.rb) via [RouteMaster](https://github.com/HouseTrip/routemaster/) for the appropriate [PropertiesTopic](https://github.com/HouseTrip/HouseTrip-Web-App/blob/master/lib/route_master/properties_topic.rb). The [Property Search Service](https://github.com/HouseTrip/property_search/) is a consumer of that event so that it can [synchronise](https://github.com/HouseTrip/property_search#search-service-syncing-overview) the new data.
```

Comment:
if you remove it then it might be a better idea to rebase it on top of this commit and force push, instead of new commit :)